src/mainframe/models/schemas.py

```diff
@@ -1,6 +1,9 @@
+from datetime import datetime
 from typing import Optional
 
-from pydantic import BaseModel
+from pydantic import BaseModel, field_serializer
+
+from src.mainframe.models.orm import Scan
```

Comment:
`src` is not a module that can be imported. This should also fix the docs CI failing
```suggestion
from mainframe.models.orm import Scan
```